src/app/modules/submission/encrypt-submission/encrypt-submission.service.ts

```diff
@@ -530,3 +534,31 @@ export const performEncryptPostSubmissionActions = (
       })
     })
 }
+
+export const getQuarantinePresignedPostData = (
+  attachmentSizes: Record<string, number>,
```

Comment:
suggestion: we could be more specific about the type, e.g.

```
type AttachmentSizesDto = {
id: string, 
contentLength: number
}[]
```

Reply:
Thanks for the suggestion, done in [447b6ab](https://github.com/opengovsg/FormSG/pull/6685/commits/447b6abfcc83cf79ee8ca232155df9bb0a15fde7) 👍